docs/index.rst

```diff
@@ -0,0 +1,54 @@
+.. hfst-optimized-lookup documentation master file, created by
+   sphinx-quickstart on Mon Feb  8 13:50:23 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+hfst-optimized-lookup for Python
+================================
+
+A pip-installable library version of hfst-optimized-lookup_, originally
+built for itwêwina_.
```

Comment:
I am worried about duplication between this file and `README.md`. Is there some way to include `README.md` here, or to write a script to keep the two in sync?

A quick web search also turns up https://recommonmark.readthedocs.io/en/latest/auto_structify.html for keeping more of this in Markdown, but I don’t know if it how common or maintained that is; I prefer Markdown but am willing to (re-?)learn `.rst` syntax.

Reply:
Haha, I feel like I _always_ have to relearn `.rst` syntax. I'll have a look!